xdf2fieldtrip.m

```diff
@@ -116,6 +152,7 @@
   data{i}.label = hdr.label;
   data{i}.time = {streams{i}.time_stamps};
   data{i}.trial = {streams{i}.time_series};
+  data{i}.event = event;
```

Comment:
This is not where other functions would the events. In case the event structure is passed from the workspace into other FieldTrip functions, if is specified as `cfg.event`. But it is never part of the data. See https://github.com/fieldtrip/fieldtrip/blob/master/utilities/ft_datatype_raw.m for the specification of this data structure.

Better would be to extend the API that is exposed to the user from 
```
data = xdf2fieldtrip(filename, ...)
```
into 
```
[data, event] = xdf2fieldtrip(filename, ...)
```